src/main/java/com/traveltime/sdk/dto/requests/proto/OneToMany.java

```diff
@@ -8,6 +8,7 @@
 @Value
 @Builder
 @AllArgsConstructor
+@With
```

Comment:
Does this do anything?

Reply:
Generates `withField` methods that create a copy of the object with the `field` set to whatever you provide in the argument. In this case I need `withDestinationCoordinates`